gcp-metric-alarm/variables.tf

```diff
@@ -0,0 +1,48 @@
+variable "alarm_configuration" {
+  type = object({
+    metric_type        = string
+    resource_type      = string
+    threshold          = number
+    duration_s         = optional(number, 60)
```

Comment:
TF 1.3.x

Reply:
(Which we can now run!)

Reply:
What's the 2nd value within `optional(number, 60)`? Is it a multiplier?

Reply:
Good q, [here](https://developer.hashicorp.com/terraform/language/expressions/type-constraints#optional-object-type-attributes) are the docs but basically the second value is the `defautl`.